core/src/bin/run.ts

```diff
@@ -70,14 +77,14 @@ export class RunCLI extends CLI {
 
     if (params.runAllSchedules) await this.runNonRecurringSchedules();
 
-    return false;
+    return true;
```

Comment:
The return method of a CLI command is `toStop` - you want to keep it as `false` or else the process will exit after this method